pages/product.js

```diff
@@ -0,0 +1,52 @@
+const { I } = inject();
+const BUY_PRODUCT = "http://opencart.qatestlab.net/index.php?route=product/product&product_id=47&page=2"
+let product_price = '';
+let option_price = '';
+let total_price = '';
+let shipping_price = '';
```

Comment:
move these variables with `let` to scenario. use `return await I.grab...()` in method, e.g. `getProductPrice()` to get price